test/shopify.test.js

```diff
@@ -215,4 +215,44 @@ describe('Shopify', () => {
         .then(res => expect(res).to.deep.equal({}));
     });
   });
+
+  describe('Shopify#callLimits', () => {
```

Comment:
It's fine, I'm picky too. :)

I moved the first callLimits test (for undefined values) to the very beginning (to assure that no requests are done before this check), and the rest to the bottom. I also improved the wording of those three tests.